Vagrantfile

```diff
@@ -67,21 +81,9 @@ Vagrant.configure("2") do |config|
     end
   end
 
-  if File.directory?("../ooni-backend")
-    config.vm.provision "shell", inline: <<-EOF
-      echo "To run oonibackend:"
-      echo "1. vagrant ssh backend"
-      echo "2. vi /etc/oonibackend.conf  # possibly"
-      echo "3. cd /data/ooni-backend"
-      echo "4. sudo ./bin/oonib -c /etc/oonibackend.conf"
-    EOF
+  config.vm.define "testing" do |testing|
+    testing.vm.network "forwarded_port", guest: 8842, host: 8142
+    testing.vm.synced_folder ".", "/data/ooni-probe"
   end
```

Comment:
I should probably delete this, I committed it by mistake. I was using the testing box as a machine I would snapshot and attempt installs on a clean system and on a system with a particular setup.